static/self.css

```diff
@@ -6,7 +6,7 @@ button,
 input,
 select,
 textarea {
-    color: #222;
+    color:  #222;
```

Comment:
`*.css` should be ignored

Reply:
Not sure where this space came from. Autopep8 will by default only run on .py files. Think I might have messed up while merging some edits made in between my update and creating the PR. Can't really prevent having at least one error like this when you edit 100+ files ;)